mh/pat_item.py

```diff
@@ -79,6 +79,61 @@ def unpack_byte(data, offset=0):
     return value
 
 
+def pack_extra_info(info):
```

Comment:
For this data struct name I was thinking something like `IndexSet` or `IndexedValues`, what do you thing?

Reply:
That's better than extra info, indeed. That gave me other naming idea, what about `ValueSet`, `FieldSet` or `optional_fields`? I'd tend to prefer `optional_fields` as the structure is similar to regular PAT's fields with an additional boolean flag.

Reply:
Indeed `optional_fields` is a good fit.